docs/guide/interactions/checks.rst

```diff
@@ -0,0 +1,368 @@
+- Then, the callback for the context menu is created.
+
+.. _guide_interaction_checks_permission-check:
+
+Permission Checks
+------------------
+
+These checks handle validating whether the user activating a command has certain permissions in a server, or whether the bot user has specific permissions.
+
+.. code-block:: python3
+    :emphasize-lines: 3,9
+
+    # User permissions check.
+    @app_commands.command()
+    @app_commands.has_permissions(administrator=True)
+    async def user_permission_check(interaction: discord.Interaction):
+        await interaction.response.send_message('Hello administrator!', ephemeral=True)
+
+    # Bot permissions check.
+    @app_commands.command()
+    @app_commands.bot_has_permissions(manage_messages=True, manage_guild=True)
+    async def bot_permission_check(interaction: discord.Interaction):
+        await interaction.response.send_message('I can do that!', ephemeral=True)
+
+These two checks are identical in usage. The only difference is that one checks the permissions of the user who calls the command, and one checks the permissions of the bot user the command is issued to.
+
+The permissions given to the check can be any number of permissions, which can be found in :class:`.Permissions`, and must have a boolean value.
+
+.. _guide_interaction_checks_cooldown-check:
+
+Cooldown Checks
+----------------
+
+It is also possible to use a check decorator that will directly attach a cooldown to your command.
+
+There are two cooldown methods, static and dynamic.
+
+Static Cooldown
+~~~~~~~~~~~~~~~~
+
+Static cooldowns are methods which limit a command to a certain number of uses per time frame.
+
+.. code-block:: python3
+    :emphasize-lines: 2
+
+    @app_commands.command()
+    @app_commands.checks.cooldown(1, 5.0, key=lambda interaction: (i.guild_id, i.user.id))
+    async def cooldown_check(interaction: discord.Interaction):
+        await interaction.response.send_message('Not on cooldown yet!', ephemeral=True)
+
+The :func:`@cooldown <.app_commands.checks.cooldown>` decorator takes 3 possible arguments:
+
+- The ``rate`` parameter first takes an integer value. This value is the number of usages that can be used within a time frame before the cooldown is activated.
+- The ``per`` parameter then takes a float value. This is the number of seconds to wait for a cooldown once it has been activated.
+- The third and final parameter, ``key``, can be used to specify what criteria are used for applying the cooldown. This is given as a function that takes a :class:`.Interaction` as an argument, and can be a coroutine.
+
+Now that we know what the parameters for a static cooldown are, let's take a closer look at the shown example:
+
+- The ``rate`` is set to ``1``, so the cooldown will trigger every time the command is used.
+- The ``per`` param is set to ``5.0``, so the cooldown will lock the command for 5 seconds when it is used.
```

Comment:
```suggestion
- The ``per`` parameter is set to ``5.0``, so the cooldown will lock the command for 5 seconds when it is used.
```
since you are already using the word parameter in the other sentences, might as well keep it consistent just to be clear to the reader. even though it's me nitpicking, this is just me going through it and making sure it's consistent and checking for flow